### DISCUSSION TITLE
Discrepancy between farm area calculated using ‘cut-plane’ function and actual farm area

### DISCUSSION THREAD

**Discussion by leilalegris** [NatLabRockies/floris#486] [Q&A]
I am having trouble with using the cut-plane function to calculate the average hub-height wind speed across the total farm area. The area calculated by the function is 15% larger than the true farm area. Are there any suggestions for how to fix this issue or any other ways to calculate the farm-averaged wind speed?

**Comment by paulf81** [comment]
If you are using the function calculate_horizontal_plane, you have the option to pass in x_bounds and y_bounds.  These default to None, and in that case they are set within grid.py (as you obsevered) like this:

```
if self.x1_bounds is None:
    self.x1_bounds = (np.min(x) - 2 * max_diameter, np.max(x) + 10 * max_diameter)

if self.x2_bounds is None:
    self.x2_bounds = (np.min(y) - 2 * max_diameter, np.max(y) + 2 * max_diameter)
```
